pages/about.py

```diff
@@ -0,0 +1,118 @@
+                        ]
+                    )
+                ),
+                html.Br(),
+                html.H3('Event Simulation', id='simulation-flow', style={'font-weight': 'bold'}),
+                html.Div('The simulation primarily tracks the series of events that will occur throughout the process of charging a vehicle. Critical parameters such as driving speed, balking probability due to long queues, and '
+                         'duration of charging stall service time were assumed based on historical data. Future steps involve large-scale SimPy execution in Python'),
+                html.Br(),
+                html.Div(
+                    style={'display': 'flex', 'justify-content': 'center', 'align-items': 'center'},
+                    children=html.Img(
+                        src='assets/simulation-flow_chart.png',
+                        style={'max-height': '250px', 'align-items': 'center'}
+                    )
+                ),
+                html.Br(),
+                html.H3('System Flow Diagram', id='system-flow', style={'font-weight': 'bold'}),
+                html.Div('The following diagram illustrates the relationship between the tools, and the FAQ section below could address some of the questions you might have. For each model, certain features will be necessary. For all features, if future projections cannot be obtained then they will have to be forecasted.'),
+                html.Div(
+                    style={'display': 'flex', 'justify-content': 'center', 'align-items': 'center'},
+                    children=html.Img(src='assets/final_system.png', style={'align-items': 'center'}),
+                ),
+                html.Br(),
+                html.Br(),
+                html.H3('Frequently Asked Questions', id='faq', style={'font-weight': 'bold'}),
+                html.Div([
+                    html.H5('1. What is Integer Programming (IP)?'),
+                    html.P('IP is a mathematical optimization technique where decision variables are restricted to be integers. Link to our IP problem can be found here:'),
+                    html.Div(
+                        [
+                            html.A("Link to our IP problem", href="https://uofwaterloo.sharepoint.com/:b:/s/tm-arts-23-24mgtecapstone-Team10/EWrFsD7LvMlDmAouvAOTAO0BRn96RlvSks6X56_Q0UrURQ?e=5U9hnl")
+                        ]),
+                    html.Br(),
+
+                    html.H5('2. What is M/M/C model in queuing theory?'),
+                    html.P('An M/M/c model represents a queueing system that has exponential arrivals, c servers with exponential service time'),
+
+                    html.H5('3. What do the charging ports mean?'),
+                    html.P('The 2, 4, 8 charging ports in the simulation refers to the number of physical connectors or outlets available for vehicles to plug into and charge simultaneously at a charging station.'),
```

Comment:
```suggestion
                    html.P('The program allows both the user and/or the optimization model to select either 2, 4, or 8 charging ports for any given charging station. This refers to the number of physical connectors or outlets available for vehicles to plug into and charge simultaneously at a charging station.'),
```
Accuracy and Clarity